Sources/AsyncHTTPClient/HTTPClient.swift

```diff
@@ -937,6 +939,7 @@ public struct HTTPClientError: Error, Equatable, CustomStringConvertible {
     /// Request contains invalid identity encoding.
     public static let identityCodingIncorrectlyPresent = HTTPClientError(code: .identityCodingIncorrectlyPresent)
     /// Request contains multiple chunks definitions.
+    @available(*, deprecated)
```

Comment:
We should add a message here like: "AsyncHTTPClient now silently corrects invalid headers."